deps/rabbit/src/rabbit_priority_queue.erl

```diff
@@ -689,6 +691,28 @@ find_head_message_timestamp(BQ, [{_, BQSN} | Rest], Timestamp) ->
 find_head_message_timestamp(_, [], Timestamp) ->
     Timestamp.
 
+find_oldest_message_received_timestamp(BQ, BQs) ->
+    %% Oldest message timestamp among all priority queues
+    Timestamps =
+        lists:foldl(
+          fun({_, BQSN}, Acc) ->
+                  case oldest_message_received_timestamp(BQ, BQSN) of
+                      '' -> Acc;
+                      Ts -> [Ts | Acc]
+                  end
+          end, [], BQs),
+    case Timestamps of
+        [] -> '';
```

Comment:
Should we return `undefined` and not an empty atom when the queue has no messages?

Reply:
It returns an empty atom to follow the same format as `head_message_timestamp` and any other empty values returned by `rabbit_amqqueue:info`.

Reply:
OK, I have suspected it might have been for consistency with certain decisions from years ago.